src/textual/app.py

```diff
@@ -406,6 +408,12 @@ def __init__(
         self.set_class(self.dark, "-dark-mode")
         self.set_class(not self.dark, "-light-mode")
 
+    def validate_title(self, title: Any) -> str:
```

Comment:
Docstrings por favor

Reply:
I tried adding docstrings _and_ making the validate methods private but that doesn't work.
I we had a similar issue recently but maybe we only added the ability for watcher methods to be private.
I think validators and compute methods should have the ability to also be private, for exactly the same reasons.
What do you think, @willmcgugan?

Reply:
> maybe we only added the ability for watcher methods to be private.

Correct, see #2382 and #2442